Source/WebCore/platform/network/curl/CurlStreamScheduler.h

```diff
@@ -38,7 +38,7 @@ class CurlStreamScheduler {
     CurlStreamScheduler();
     virtual ~CurlStreamScheduler();
 
-    WEBCORE_EXPORT CurlStreamID createStream(const URL&, CurlStream::Client&, CurlStream::ServerTrustEvaluation = CurlStream::ServerTrustEvaluation::Enable);
+    WEBCORE_EXPORT CurlStreamID createStream(const URL&, CurlStream::Client&, CurlStream::ServerTrustEvaluation, bool);
```

Comment:
>  Usually, there should be a parameter name for bools, strings, and numerical types.
https://webkit.org/code-style-guidelines/#names-variable-name-in-function-decl
`bool enableLocalhostAlias`